src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/benificialowner/psc/Psc.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.benificialowner.psc;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.LocalDate;
+import java.util.Objects;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.NatureOfControl;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.Address;
+
+public abstract class Psc {
+  @JsonProperty("actionDate")
+  private LocalDate actionDate = LocalDate.now();
+
+  @JsonProperty("serviceAddress")
+  private Address serviceAddress;
+
+  @JsonProperty("residentialAddress")
+  private Address residentialAddress;
+
+  @JsonProperty("natureOfControls")
+  private NatureOfControl natureOfControls = new NatureOfControl();
```

Comment:
Since it's a single object and not a list
```suggestion
  private NatureOfControl natureOfControl = new NatureOfControl();
```

Reply:
So actually the Class NatureOfControl actually contains a list of Strings that represent many NatureOfControls

Looking into this, would it be better to rename the class as plural rather than the variable?